images/init-syncthing/run.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
```

Comment:
That's different. I'm talking about a test to validate the script is doing what we think it should do. It's very easy to change something and end with syntax errors or a bad check.

 Since we're adding functionality on this pr, the test should be here are well